src/erlang_ls.app.src

```diff
@@ -29,6 +30,7 @@
         %% Logging
       , {logging_enabled, false}
       , {log_dir, "/usr/local/var/log/erlang_ls"}
+      , {db_dir, "/usr/local/var/cache/erlang_ls"}
```

Comment:
Will this work on different operating systems?

Reply:
Fair point. The logging would also break, but that's disabled by default.
On the other hand, if we aim at supporting multiple platforms we should somehow add them to the CI pipelines.

Reply:
[`filename:basedir/3`](http://erlang.org/doc/man/filename.html#basedir-3) might be useful to get correct temp or config directories on various systems

Reply:
Nice! I didn't know about it.

Reply:
That's what happens when you tend to only support one system :p